src/router/index.ts

```diff
@@ -42,6 +42,14 @@ export default route<StateInterface>(function (/* { store, ssrContext } */) {
         const preferredChainName = configMgr.getPreferredChain();
 
         if (to.path === '/') { // if attempting to go to home page
+            if (process.env.SHOW_MULTICHAIN_SELECTOR !== 'true') {
+                return({
+                    path: 'network',
+                    query: {
+                        network: process.env.CHAIN_NAME,
+                    },
+                });
+            }
             if (to.query.network) { // if there is network param, proceed to network with the param
```

Comment:
@jaegerfe If multichain selector is set to false, it will never reach network param logic, is this the intended behavior or just a temporary fix?

Reply:
@donnyquixotic Yes, it was supposed to be a quick and dirty fix :)
I added a new commit that I believe addresses both #766 and #767. Please, double check and let me know!

@jaegerfe it would be awesome if you could also review it ❤️

Reply:
@donnyquixotic I’ve updated the PR once again - and I believe this is the last time. Eduardo pointed that I had left some unreachable condition, so I just removed that for now.

Also added @poplexity as a reviewer since it doesn't make sense that I approve my own changes.